geode-core/src/main/java/org/apache/geode/cache/query/security/JavaBeanAccessorMethodAuthorizer.java

```diff
@@ -167,7 +167,7 @@ public boolean authorize(Method method, Object target) {
    *
    * @return an unmodifiable view of the allowed packages for this authorizer.
    */
-  Set<String> getAllowedPackages() {
+  public Set<String> getAllowedPackages() {
```

Comment:
Any reason why you changed the visibility to `public`?.

Reply:
To bring it in line with the RegExMethodAuthorizer, and to help facilitate DUnit testing.

Reply:
Perfect, thanks!.